src/lib/button-toggle/button-toggle.ts

```diff
@@ -230,41 +218,89 @@ export class MatButtonToggleGroup extends _MatButtonToggleGroupMixinBase
    */
   setDisabledState(isDisabled: boolean): void {
     this.disabled = isDisabled;
-    this._markButtonTogglesForCheck();
-  }
 
-  private _markButtonTogglesForCheck() {
     if (this._buttonToggles) {
-      this._buttonToggles.forEach((toggle) => toggle._markForCheck());
+      this._buttonToggles.forEach(toggle => toggle._markForCheck());
     }
   }
-}
 
-/** Multiple selection button-toggle group. `ngModel` is not supported in this mode. */
-@Directive({
-  selector: 'mat-button-toggle-group[multiple]',
-  exportAs: 'matButtonToggleGroup',
-  inputs: ['disabled'],
-  host: {
-    'class': 'mat-button-toggle-group',
-    '[class.mat-button-toggle-vertical]': 'vertical',
-    'role': 'group'
+  /** Dispatch change event with current selection and group value. */
+  _emitChangeEvent(): void {
+    const selected = this.selected;
+    const source = Array.isArray(selected) ? selected[selected.length - 1] : selected;
+    const event = new MatButtonToggleChange(source, this.value);
+    this._controlValueAccessorChangeFn(event.value);
+    this.change.emit(event);
   }
-})
-export class MatButtonToggleGroupMultiple extends _MatButtonToggleGroupMixinBase
```

Comment:
This would be a major change since it totally removes the symbol; people may have been using `@ViewChild` etc. to grab hold of these. Can we leave the symbol in as an alias for the combined one?